develop-docs/sdk/telemetry/traces/index.mdx

```diff
@@ -228,16 +228,22 @@ A transaction's sampling decision should be passed to all of its children, inclu
-  1. This also applies to Tracing without Performance where we simply want to generate `sample_rand` every time a new trace is started. This `sample_rand` value can then be stored on `PropagationContext`.
-1. It is _recommended_ to generate the random number deterministically using the trace ID as seed or source of randomness (make sure to do this atomically so multiple threads accessing the same random instance don't mix their seeds). The exact method by which the random number is created is implementation-defined and may vary between SDK implementations.
-1. The `sample_rand` is part of the DSC (Dynamic Sampling Context) and as with other values on the `baggage` header, the `sample_rand` value from the current execution context should be propagated to downstream SDKs. It should also be sent to other system as part of the `baggage` header if Performance is disabled and sampling decision is deferred.
-1. On incoming traces, an SDK takes the incoming `sentry-sample_rand` value in the `baggage` header and uses it for the rest of the current execution context (for example, request) by storing it in the `PropagationContext`.
-1. If `sample_rand` is missing on an incoming trace, the SDK creates a new one applying **special rules** and uses it for the current execution context:
-   1. If `sample_rate` (inside `baggage`) and the sampling decision (trailing `-1` or `-0` from the `sentry-trace` header) are present in the incoming trace, create `sample_rand` so that when compared to the incoming `sample_rate` it would lead to the same sampling decision that is in the `sentry-trace` header. This means, for a decision of `True` generate a random number in half-open range `[0, rate)` and for a decision of `False` generate a random number in range `[rate, 1)`.
-   1. If the sampling decision is missing on the incoming trace, generate a random number in range of `[0, 1)` (including 0.0, excluding 1.0), like for a new trace.
+SDKs must set and propagate the `sample_rand` as follows:
+
+1. Whenever a new trace is started, including in the Tracing without Performance case, the SDK generates a `sample_rand` value for the **entire trace**.
+   1. SDKs must generate the `sample_rand` by pseudo-randomly sampling a floating-point number from the uniform distribution over the range `[0.0, 1.0)` (including `0.0`, excluding `1.0`).
+   1. The exact method for pseudo-random sampling is implementation-defined, and may therefore vary between SDK implementations.
+   1. We _recommend_ that SDKs generate the random number deterministically by using the trace ID to seed the random number generator. If multiple threads share the same random instance, seed the random number generator atomically, so that seeds do not get shared accross threads.
+1. The `sample_rand` is part of the Dynamic Sampling Context (DSC). Therefore, SDKs must propagate the `sample_rand` to downstream SDKs by adding it to the `baggage` header.
```

Comment:
Would love some input from SDK folks here, since the previous spec already mentioned `baggage`:

I would not call out baggage explicitly. There's already gotta be a place where we define how the DSC is supposed to be propagated and we shouldn't need to duplicate this mechanism here. However, if it helps to mention `baggage` for clarity, I'm open to keep it.

Same for the sub-bullet on how to propagate sample_rand in baggage. Shouldn't we have central documentation for this on the [DSC page](https://develop.sentry.dev/sdk/telemetry/traces/dynamic-sampling-context/)?